### ISSUE TITLE
Cache not flushed when deleting product

### ISSUE THREAD

**Issue by figinternet** [GordonLesti/Lesti_Fpc#111]
When Disabling a product in magento, the category page is flushed.
But when a product is deleted the category page is not flushed from the cache,

The workaround for this problem for now is first disabling the product, or re-save the product, so the category page is flushed.

It looks for me like a Bug. and i think it's simmilar to #67

**Comment by Mic2005** [comment]
Great if you manually can select what sort of fpc cache you want to flush ;)
Op 14 apr. 2015 12:24 schreef "figinternet" <notifications@github.com>:

**Comment by figinternet** [comment]
@Mic2005 I'ts not a manual action but the automatic flushing of all the pages related with the product.
the product page self is flushing and the category page is also flushing on changing a product.
But when deleting it, the category page's are not flushing.

**Comment by figinternet** [comment]
@GordonLesti Do you have any solution for this bug/problem ?

**Comment by GordonLesti** [comment]
@figinternet yes, in my mind, but I can't find enough time. I hope to solve this on the next hackathon.

**Comment by figdeveloper** [comment]
@GordonLesti When is the next hackathon ? 
I love to see some kind of solution
